src/main/java/bio/terra/common/stairway/StairwayComponent.java

```diff
@@ -151,6 +187,10 @@ public Stairway get() {
   }
 
   private String getClusterName() {
+    if (stairwayProperties.getClusterNameSuffix() == null) {
+      throw new IllegalArgumentException(
+          "Cluster name suffix property is required for Stairway work queue creation");
```

Comment:
nit: I think this assertion might be clearer if it were inlined in setupWorkQueue. As-is, I wouldn't really expect this validation to happen in a method named `getClusterName`, and I initially thought the comment on setUpWorkQueue was incorrect.